hoodie-client/src/test/java/com/uber/hoodie/index/TestHbaseIndex.java

```diff
@@ -359,6 +359,23 @@ public void testsHBasePutAccessParallelism() {
     Assert.assertEquals(11, hbaseNumPuts);
   }
 
+  @Test
```

Comment:
Can we also add a test case for triggering index.updateLocation twice and validate the correctness after persisting the RDD's ?

Reply:
Added `testTagLocationAndDuplicateUpdate` in TestHbaseIndex, pls take a look.